server/Main.hs

```diff
@@ -154,7 +159,7 @@ tryParseType = hush . fmap (CST.convertType "<file>") . runParser CST.parseTypeP
 
     runParser :: CST.Parser a -> Text -> Either String a
     runParser p =
-      first (CST.prettyPrintError . NE.head)
+      bimap (CST.prettyPrintError . NE.head) snd
```

Comment:
As above, we now have a tuple with `[CST.ParserWarning]`; I opted to throw these warnings away, following the lead of `Language.PureScript.Interactive.Parser`:

https://github.com/purescript/purescript/blob/7ecc42669c69682996f2196ba2eef6c4ca827348/src/Language/PureScript/Interactive/Parser.hs#L66-L71